devops/providers/azure-devops/templates/infrastructure/README.md

```diff
@@ -0,0 +1,204 @@
+# Infrastructure Continuous Integration and Deployment Pipeline
+
+Azure Pipeline build and release templates responsible for operationalizing infrastructure resources with a focus on repeatability and test automation.
+
+## Introduction
+
+Azure Devops is the common fabric between developers, process and products to enable continuous delivery of value to customers.
+
+Cobalt's devops release flow follows a git commit driven model. Meaning that azure pipelines automatically builds, validates and release Cobalt templates every pull request and commit to your source git repository, essentially applying the GitOps pattern to the Azure PaaS services.
+
+This document outlines how Cobalt's CI/CD Azure Pipeline templates will be used to meet the devops related use-cases for deployment operators working on Cobalt.
+
+## Deployment Topology
+
+This graphic shows the CI/CD workflow topology needed by our enterprise customers to deploy infrastructure to Azure. The journey starts with a git commit to the IAC repo. This workflow validates and deploys the desired infrastructure state across the azure environments.
+
+![infrastructure_reference_cicd_flow](https://user-images.githubusercontent.com/7635865/60440330-3a039580-9bda-11e9-8af6-a883739c2d87.jpg)
```

Comment:
All great comments. Please see my responses to each on of your points. 

1) This will be evaluated within the yaml definition as part of the workspace setup step in the build. I added `- Implementation details of the Azure Pipeline Template.` to the out of scope list as that will be addressed during the implementation user story. 
2) Fixed
3) Fixed by removing the merge auto trigger for the stage. 
4) Not sure but I view this as implementation details which is out of scope.